Resources/public/js/views/tabs/ez-locationviewlocationstabview.js

```diff
@@ -106,6 +109,29 @@ YUI.add('ez-locationviewlocationstabview', function (Y) {
         },
 
         /**
+         * Tap event handler on the `Swap location` button. It fires the
+         * `swapLocation` event
+         *
+         * @method _swapLocation
+         * @protected
+         * @param {EventFacade} e
+         */
+        _swapLocation: function (e) {
+            /**
+             * Fired when the user clicks on `Swap location` button
+             *
+             * @event swapLocation
+             * @param {eZ.Location} location the content to be swaped
+             * @param {Function} afterCreateCallback callback function that will be called after
```

Comment:
The main problem is that the name of the variable is wrong in the JSdoc.
Regarding the naming of the callback, I am nitpicking, both names are ok